src/components/ChordRowStrip.css

```
@@ -0,0 +1,209 @@
+  font-size: 0.75rem;
+  font-weight: var(--font-weight-medium);
+  color: var(--text-soft-white);
+  opacity: 0.65;
+  letter-spacing: 0.02em;
+  white-space: nowrap;
+}
+
+/* Responsive tweaks */
+.app-container[data-layout-tier="mobile"] .chord-row-strip {
+  --chip-size: 2.15rem;
+  width: 100%;
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)